pkg/provisioningv2/rke2/planner/etcdrestore.go

```diff
@@ -175,6 +175,9 @@ func (p *Planner) runEtcdRestoreControlPlaneEtcdServiceStop(controlPlane *rkev1.
 	servers := collect(clusterPlan, anyRole)
 	updated := false
 	for _, server := range servers {
+		if server.Machine.Status.NodeInfo.OperatingSystem == windows {
```

Comment:
Instead of doing this here, can we use a different function besides `anyRole` to the collect function above that filters out Windows nodes?

That would be cleaner, in my opinion.

Reply:
In addition, and I know this isn't part of your change, but could we change the name of the encapsulating function so that it is more descriptive of what is happening here? Specifically, we are dealing with all non-Windows machines at the moment. Not only control plane and etcd.